### PR TITLE
First draft of the peer performance review page, replacing the six month one-on-one review

### PR DESCRIPTION
**Pull request by lewisnyman**
This proposal is specifically designed for a flat heirachy team, and integrated with the buddy system.

This is inspired by the performance review model at Github: https://zapier.com/learn/the-ultimate-guide-to-remote-working/remote-employee-evaluation/

### CODE REVIEW

**Comment by lewisnyman** [comment]
Steve P feedback: 
We should avoid making it seem like a corporate appraisal or an intervention - instead of "performance" maybe something else. It could be something around nurturing growth and success.
We should make it clear be helping a person be the best they can be, figuring out the best direction for them grow in.
We should unstigmatise the idea that someone could grow out of the company and do great things elsewhere. Maybe each employee could have a personal mission statement like the company.
Review against the companies core principles. What has the employee done to further the companies mission?

File: working-at-wunderkraut/career-development/personal-growth-peer-review.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+We use peer review to provide perspective, to help us understand ourselves, how we can improve, and what we can do to help each other more.
+
+Every six months, you can ask your buddy to collect feedback from three to five co-workers of your choice. Choose co-workers who will have gained insight from working closely with you recently.
+
+Your buddy introduces your own goals to the co-workers. These can include:
+
+1. Where do you want to be in five years
+2. How can I be more open, professional, healthy, fluid, and a leader? See [our principles](/about-the-company/principles/)
+3.
```
Comment (joesb) [inline working-at-wunderkraut/career-development/personal-growth-peer-review.md]: Reviewing this, there seems to be a stray '3.' here - was there a third point to be added, or should it be removed?

**Comment by lewisnyman** [comment]
@huntons Any chance you could comment here?

**Comment by joesb** [comment]
This looks good to me. Merging - we can always iterate later.

**Pull request merged**